code/__DEFINES/nodes.dm

```diff
@@ -7,3 +7,14 @@
 //DEFINES for AI behavior to utilize to show off what it's currently doing
 #define MOVING_TO_NODE "moving_to_a_node" //Move to a node
 #define MOVING_TO_ATOM "moving_to_an_atom" //We want to move to this thing and probably hit it; can be just about anything like a mob or machinery
+
+//Weight types for nodes
+#define NODE_LAST_VISITED "node_last_visited" //Set to world.time when visited by an AI
```

Comment:
Taken in a vacuum. Will this result in a 'more spread out' style of attack from AI mobs?

Reply:
Makes it somewhat less RNG for not going to certain places and that the `NODE_LAST_VISITED` isn't updated if the AI switches from moving around to attacking an enemy.
I'll look forward into making this but re-implementing the old functionality of moving to nodes with enemies present.